src/settings/storeconf.h

```diff
@@ -22,8 +22,9 @@ class StoreConf : public QWidget
     QString applicationName         = "Pair Storm";
     QString analizerStyle           = "white";
     QString analizerFontSize        = "12";
-    QString analizerFontName        = "Times New Roman";
-    QString cppExtentions           = "c;cpp;h;hpp";
+    QString analizerFontName        = "Courier";
+    QString cppExtentions           = "c;cpp;h;hpp;json;txt";
+    QStringList cppExtentionsList {"c", "cpp", "h", "hpp", "json", "txt"};
 
```

Comment:
Please provide file extensions in format ".c" ".cpp" etc

Reply:
Done